packages/expo-updates/e2e/__tests__/fixtures/Updates-assets.e2e.js

```diff
@@ -1,28 +1,27 @@
-import path from 'path';
-import { setTimeout } from 'timers/promises';
-import uuid from 'uuid/v4';
+const path = require('path');
```

Comment:
any reasons we should convert ts to js? is it because to simplify the detox setup without something like ts-jest?

Reply:
That is the reason 😀